gender_novels/analysis/gender_pronoun_freq_analysis.py

```diff
@@ -581,17 +582,18 @@ def overall_mean(d):
     mean = np.mean(l)
     return mean
 
-
-if __name__ == '__main__':
-    # from dh_testers.testRunner import main_test
-    # main_test()
-    # print("mean relative female freq across corpus:")
-    # print(relative_frequency_overall(Corpus('sample_novels')))
+def run_all_analyses():
+    '''
+    Runs analyses for:
+        Female and Male pronoun frequency for:
+            author gender, publication date, publication, publication location
+        Female and Male Subject Object frequency Comparison for:
+            author gender, publication date, publication, publication location
+    Prints results nicely
+    :return: None
     '''
     all_data = books_pronoun_freq(Corpus('gutenberg'))
```

Comment:
save the gutenberg corpus as a variable:
```
gutenberg_corpus = Corpus('gutenberg')
```
and substitute the other calls to "Corpus('gutenberg')" with "gutenberg_corpus"
so that it doesn't need to be loaded three times.